manager/container.go

```diff
@@ -42,6 +42,7 @@ import (
 )
 
 // Housekeeping interval.
+var enableLoadReader = flag.Bool("enable_load_reader", false, "Whether to enable cpu load reader")
```

Comment:
I actually just moved that flag over from `manager.go` (i.e. it already exists in cAdvisor). 

We could definitely use `disable_metrics` to disable CPU load monitoring, but that'd break CLI compatibility with past versions.

Let me know what you think,